shared/src/constants/constants.ts

```diff
@@ -126,7 +141,7 @@ const PutThetaVault: VaultOptions[] = [
 
 export const SolanaAssets = ["SOL"];
 
-export const isSolanaVault = (vaultOption: VaultOptions) => {
+export const isSolanaVault = (vaultOption: string) => {
```

Comment:
Having it as a `string` has the nice property of being able to pass in anything into the function rather than doing:
```js
isSolanaVault(xyz as VaultOptions)
```

Does not need to be typed because it's just comparing against an array